diff -Nru FSViewer.app-0.2.3/configure.in FSViewer.app-0.2.3.new/configure.in
--- FSViewer.app-0.2.3/configure.in	Sat Oct  9 20:45:21 1999
+++ FSViewer.app-0.2.3.new/configure.in	Mon Sep 17 18:39:05 2001
@@ -5,10 +5,7 @@
 
 AC_PROG_MAKE_SET
 
-PACKAGE=FSViewer.app
-AC_SUBST(PACKAGE)
-VERSION=0.2.3
-AC_SUBST(VERSION)
+AM_INIT_AUTOMAKE(FSViewer.app, 0.2.3)
 
 dnl Checks for programs.
 AC_PROG_CC
@@ -17,11 +14,6 @@
 AC_ARG_PROGRAM
 AC_PROG_RANLIB
 
-AC_PATH_PROG(AUTOCONF, autoconf)
-AC_PATH_PROG(ACLOCAL, aclocal)
-AC_PATH_PROG(AUTOHEADER, autoheader)
-AC_PATH_PROG(AUTOMAKE, automake)
-
 dnl the prefix
 dnl ==========
 dnl
@@ -43,9 +35,6 @@
 fi
 
 dnl Additional "default" items
-#LIBS="$LIBS -lX11"
-#CFLAGS="$CFLAGS -I/usr/local/include"
-#LDFLAGS="$LDFLAGS -L/usr/local/lib"
 
 dnl Check for get-wraster-flags
 AC_CHECK_PROG(GET_WRASTER_FLAGS, get-wraster-flags, get-wraster-flags)
@@ -154,4 +143,4 @@
 
 AC_DEFINE_UNQUOTED(ICONDIR, "${fsviewerdir}")
 
-AC_OUTPUT(Makefile src/Makefile src/regexp/Makefile defs/Makefile)
+AC_OUTPUT(Makefile src/Makefile defs/Makefile)
diff -Nru FSViewer.app-0.2.3/src/Makefile.am FSViewer.app-0.2.3.new/src/Makefile.am
--- FSViewer.app-0.2.3/src/Makefile.am	Sat Oct  9 20:45:22 1999
+++ FSViewer.app-0.2.3.new/src/Makefile.am	Mon Sep 17 18:38:49 2001
@@ -1,10 +1,9 @@
 ## Process this file with automake to produce Makefile.in
 ##
-SUBDIRS                 = regexp
-bindir			= @fsviewerdir@
+bindir			= $(fsviewerdir)
 bin_PROGRAMS		= FSViewer
-datadir			= @fsviewerdir@
-pkgdatadir              = @fsviewerdir@
+datadir			= $(fsviewerdir)
+pkgdatadir              = $(fsviewerdir)
 FSViewer_SOURCES	= FSViewer.c files.c FSFileBrowser.c\
 			  list.c misc.c prop.c timestampWidget.c \
 			  attribsInspector.c editInspector.c exeInspector.c \
@@ -13,11 +12,15 @@
 			  FSFileView.c FSInspector.c magic.c DnD.c \
 			  FSPathView.c xmodifier.c FSFinder.c FSFileButton.c\
 			  FSMCList.c FSPrefs.c
-LDADD                   = -Lregexp -lregx
 noinst_HEADERS		= FSViewer.h filebrowser.h files.h list.h misc.h \
 			  timestampWidget.h FSBrowser.h FSPanel.h FSUtils.h \
 			  FSMenu.h FSFileView.h magic.h DnD.h FSFileButton.h \
 			  FSFinder.h FSMCList.h FSPathView.h FSPrefs.h
 pkgdata_DATA		= MagicFiles 
-##EXTRA_DIST		= icons/* xpm/* fsviewer.sh
-EXTRA_DIST		= xpm/* MagicFiles
+EXTRA_DIST		= MagicFiles arrow.xpm backing2.xpm backing.xpm \
+			  clk.xpm date.xpm file_plain.xpm FSViewer.xpm \
+			  led.xpm list_dir.xpm list_dir.xpm.bak \
+			  list_exe.xpm list_file.xpm month.xpm transparent.xpm \
+			  weekday.xpm wmanalog.xpm wmdigital.xpm wmtime.xpm \
+			  year.xpm
+
diff -Nru FSViewer.app-0.2.3/src/regexp/Makefile.am FSViewer.app-0.2.3.new/src/regexp/Makefile.am
--- FSViewer.app-0.2.3/src/regexp/Makefile.am	Sat Oct  9 20:45:23 1999
+++ FSViewer.app-0.2.3.new/src/regexp/Makefile.am	Thu Jan  1 01:00:00 1970
@@ -1,5 +0,0 @@
-## Process this file with automake to produce Makefile.in
-##
-libregx_a_SOURCES	= regerror.c regexp.c regsub.c regexp.h regmagic.h
-noinst_LIBRARIES	= libregx.a
-EXTRA_DIST		= README regexp.3
diff -Nru FSViewer.app-0.2.3/xpm/Makefile.am FSViewer.app-0.2.3.new/xpm/Makefile.am
--- FSViewer.app-0.2.3/xpm/Makefile.am	Fri Apr 16 19:34:30 1999
+++ FSViewer.app-0.2.3.new/xpm/Makefile.am	Mon Sep 17 18:38:28 2001
@@ -1,4 +1,36 @@
 icondatadir              = $(pkgdatadir)/icons
 
-icondata_DATA            = *.xpm *.tif
+icondata_DATA            = arrow.xpm file-dot-c.xpm file-dot-mov.xpm \
+			   file-dot-sh.xpm file-generic-gnustep.xpm \
+			   generic-library.xpm clk.xpm file-dot-dat.xpm \
+			   file-dot-mp3.xpm file-dot-so.xpm file-link.xpm \
+			   generic-sound.xpm date.xpm file-dot-deb.xpm \
+			   file-dot-mpg.xpm file-dot-tar.xpm \
+			   file-makefile.xpm generic-text.xpm \
+			   exec-link.xpm file-dot-eps.xpm file-dot-m.xpm \
+			   file-dot-tcl.xpm file_plain.xpm home.xpm \
+			   file-core.xpm file-dot-f90.xpm file-dot-o.xpm \
+			   file-dot-texi.xpm file-plain.xpm led.xpm \
+			   file-dot-aiff.xpm file-dot-fig.xpm \
+			   file-dot-pdf.xpm file-dot-tex.xpm \
+			   file-readme.xpm month.xpm file-dot-au.xpm \
+			   file-dot-for.xpm file-dot-pedro.xpm \
+			   file-dot-tga.xpm folder-link.xpm mycomputer.xpm \
+			   file-dot-avi.xpm file-dot-gif.xpm \
+			   file-dot-pl.xpm file-dot-txt.xpm folder.xpm \
+			   neighbour.xpm file-dot-a.xpm file-dot-gz.xpm \
+			   file-dot-png.xpm file-dot-wav.xpm Folder.xpm \
+			   penguin.xpm file-dot-bz2.xpm file-dot-html.xpm \
+			   file-dot-ps.xpm file-dot-xbm.xpm FSViewer2.xpm \
+			   weekday.xpm file-dot-cc.xpm file-dot-h.xpm \
+			   file-dot-py.xpm file-dot-xcf.xpm FSViewer.xpm \
+			   wmanalog.xpm file-dot-class.xpm \
+			   file-dot-jar.xpm file-dot-rc.xpm \
+			   file-dot-xpm.xpm full-folder.xpm wmdigital.xpm \
+			   file-dot-conf.xpm file-dot-java.xpm \
+			   file-dot-rpm.xpm file-dot-zip.xpm \
+			   generic-coding.xpm wmtime.xpm file-dot-cpp.xpm \
+			   file-dot-jpg.xpm file-dot-shar.xpm \
+			   file-executable.xpm generic-conf.xpm year.xpm
+
 EXTRA_DIST              = $(icondata_DATA)
